lib/web/servers.go

```diff
@@ -85,6 +85,46 @@ func (h *Handler) clusterDatabasesGet(w http.ResponseWriter, r *http.Request, p
 	}, nil
 }
 
+// clusterDatabaseGet returns a list of db servers in a form the UI can present.
+func (h *Handler) clusterDatabaseGet(w http.ResponseWriter, r *http.Request, p httprouter.Params, ctx *SessionContext, site reversetunnel.RemoteSite) (interface{}, error) {
+	databaseName := p.ByName("database")
+	if databaseName == "" {
+		return nil, trace.BadParameter("database name is required")
+	}
+
+	clt, err := ctx.GetUserClient(site)
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+
+	db, err := clt.GetDatabase(r.Context(), databaseName)
```

Comment:
As discussed, we should be fetching database servers here.

Reply:
Changing this to DatabaseServers

cc @greedy52

Reply:
@r0mant Can you please take another look?
I'm now using DatabaseServers as source